css/pages/index.css

```diff
@@ -3,50 +3,122 @@
 }
 
 .container {
-  width: 750px;
+  width: 100%;
   text-align: center;
   font-size: 1.5rem;
 }
 
+.second-screen {
+  width: 100%;
+}
+
 .cards-title {
-  margin-bottom: 32px;
-  color: #e20665;
+  margin-top: 1041px;
+  margin-bottom: 60px;
+  width: 100%;
+}
+
+h3 {
+  color: #091133;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  letter-spacing: 10%;
+  font-weight: 600;
+  font-size: 250%;
+  line-height: 66%;
+  font-style: normal;
+  font-family: 'raleway';
 }
 
 .cards {
   display: flex;
-  justify-content: space-around;
+  justify-content: center;
+  width: 100%;
+  margin-left: 5%;
 }
 
 .card {
   border: 1px solid #f3efef;
-  box-shadow: 0 0 15px -9px rgba(0, 0, 0, 0.65);
-  border-radius: 2px;
-  width: 30%;
-  padding: 18px 15px;
+  border-radius: 18px;
   display: flex;
   justify-content: center;
   align-items: center;
   font-weight: 600;
   font-size: 0.8em;
   text-align: center;
-  color: #000000;
+  letter-spacing: 1px;
+  background-color: #e30062;
+  color: #ffffff;
+  font-family: 'Roboto';
+  font-style: normal;
+  padding: 3%;
+  margin-right: 7%;
+}
+
+#card1 {
+  width: 18%;
+}
+
+#card2 {
+  width: 18%;
+}
+
+#card3 {
+  width: 18%;
 }
 
 .card:hover {
   box-shadow: 0 0 20px -8px rgba(0, 0, 0, 0.65);
-  color: #000000;
 }
 
 .card:active {
   background-color: #fff;
   color: #929292;
 }
 
-.social-handles {
+.info-note-div {
+  margin-top: 5%;
+  background-color: #f7f4f4;
+  padding: 7% 10%;
+  width: 100%;
+  display: flex;
+}
+
+.info-note-span {
+  flex: 50%;
+  /* padding: 5px; */
+}
+
+.info-note {
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: bold;
+  font-size: 35px;
+  letter-spacing: 1px;
+  color: #091133;
+  line-height: 55px;
+  text-align: left;
+  width: 50%;
+  margin-top: 5%;
+}
+
+.info-note-link {
+  color: #e30062;
+}
+
+.info-note-img {
+  float: right;
+  width: 80%;
+  display: flex;
+  margin-top: -10%;
```

Comment:
Negative margin.... 🤔
Where is this being used and for what purpose?

Reply:
It is used for that mobile marketing image on the right to make it slightly upwards as compared to the info paragraph.